src/Service/Gateway.php

```diff
@@ -11,6 +11,7 @@
 use Shopware\Core\Framework\DataAbstractionLayer\EntityRepositoryInterface;
 use Shopware\Core\Framework\Validation\DataBag\RequestDataBag;
 use Shopware\Core\System\SalesChannel\SalesChannelContext;
+use Shopware\Storefront\Controller\ErrorController;
```

Comment:
ErrorController is not needed in Gateway.php

Reply:
https://github.com/emspay/ems-online-shopware-6/commit/ad46c734d3e80c6b3224b676a128be6cb4326a6e

Reply:
There are still references to ErrorController in this file and in services.xml that need to be removed too.

Reply:
@volgustleasoft done